frontend/src/metabase/home/components/Onboarding/OnboardingDismissedToast.tsx

```diff
@@ -0,0 +1,16 @@
+import { t } from "ttag";
+
+import { Stack, Text, Title } from "metabase/ui";
+
+export const OnboardingDismissedToast = () => {
+  return (
+    <Stack spacing="xs">
+      <Title
+        order={4}
+        color="white"
+        fw={700}
+      >{t`Page hidden from the navigation sidebar`}</Title>
+      <Text color="white">{t`Access it later anytime by clicking the gear icon at the top right of the screen.`}</Text>
```

Comment:
A context string explaining that 'it' is a page will help the translators get the grammar right.

Reply:
https://github.com/metabase/metabase/pull/50536/commits/fdd4fa0da8494e5b347477ac02fb2a8d8b456a47